apps/backend/src/database/database.service.ts

```diff
@@ -12,9 +12,11 @@ export class DatabaseService {
   }
 
   async cleanAll(): Promise<void> {
-    Object.values(this.sequelize.models).forEach((model) => {
-      model.destroy({where: {}});
-    });
+    const promises: Promise<number>[] = [];
+    for (const model of Object.values(this.sequelize.models)) {
```

Comment:
could be simplified by just doing a map against the object.values

i.e. something like

await Promise.all(Object.values(this.sequelize.models).map(model => model.destroy(blah)))

Reply:
Good suggestion. I disregarded `.map` because it also is not able to handle promises gracefully.